docs/index.rst

```diff
@@ -113,7 +113,7 @@ function. For example::
             flask.flash('Logged in successfully.')
 
             next = flask.request.args.get('next')
-            # next_is_valid should check if the user has valid 
+            # next_is_valid should check if the user has valid
```

Comment:
There's a lot changed in this file that doesn't seem relevant to this patch. It makes it difficult to evaluate the patch on its own and I don't generally think it's good practice to merge commits with irrelevant changes.